dropwizard-testing/src/main/java/io/dropwizard/testing/DropwizardTestSupport.java

```diff
@@ -238,6 +240,7 @@ public void after() {
             stopIfRequired();
```

Comment:
This calls `configuration.getLoggingFactory().reset()` which does

```
loggerContext.stop();
final Logger logger = loggerContext.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
logger.detachAndStopAllAppenders();
```

so in theory the appenders should already be stopped in `after()`?

Reply:
Good point. But this only holds, if the configuration is set. When debugging the test, the configuration is sometimes `null`. The `Application` calls `bootstrapLogging()` anyway. Therefore we should ensure, that the appenders are stopped, if the configuration was not provided.